CHANGELOG.md

```diff
@@ -2,10 +2,15 @@
 
 Version history.
 
+## 0.0.9 released 2022-07-29
+
+- New option `--tab` to set tab-size or keep tabs.
+- Tested with GHC 8.0.2 - 9.2.3 and 9.4.1 alpha.
```

Comment:
GHC 9.2.4 is just released. Can you test it? 

I am not sure if it is a good idea to include an alpha version here, as those releases are mainly for developers.

Reply:
I tested compilation now with 9.2.4 and 9.4.1.